src/pages/index.js

```diff
@@ -70,31 +155,38 @@ function createCard(data) {
       handleImageClick: () => {
         imageModal.open(data);
       },
+      handleLikeClick: (card) => {
+        api
+          .likeCardStatus(card.getId(), !card._isLiked)
+          .then((data) => {
+            card.handleLike(data.isLiked);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+      },
+      handleDeleteCardClick: confirmDeleteCard,
     },
     cardSelectors.cardTemplate,
   );
   return card.getView();
 }
 
-// /* Event Handlers */
-
-function handleProfileEditFormSubmit(profileData) {
-  const name = profileData.title;
-  const bio = profileData.bio;
-  user.setUserInfo({ name, bio });
-  editProfileModal.close();
+function confirmDeleteCard(cardData) {
+  confirmDeleteModal.open();
+  confirmDeleteModal.confirmDelete(() => {
+    api
+      .deleteCard(cardData.getId())
+      .then(() => {
+        cardData.handleDeleteCard();
+        confirmDeleteModal.close();
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors